go/store/val/tuple_descriptor.go

```diff
@@ -669,3 +670,53 @@ func (td TupleDesc) Equals(other TupleDesc) bool {
 	}
 	return true
 }
+
+type AddressTypeHandler struct {
+	vs           ValueStore
+	childHandler TupleTypeHandler
+}
+
+func NewExtendedAddressTypeHandler(vs ValueStore, childHandler TupleTypeHandler) AddressTypeHandler {
+	return AddressTypeHandler{
+		vs:           vs,
+		childHandler: childHandler,
+	}
+}
+
+func (handler AddressTypeHandler) SerializedCompare(ctx context.Context, v1 []byte, v2 []byte) (int, error) {
```

Comment:
Not really a criticism, but I'm not sure this is in the spirit of this method. I.e. if you have to deref an address to get the value to compare, this is probably going to be very slow, and the point of this method is to be faster than deserializing the values just to compare them.

Not really sure what to do about it though.